### PR TITLE
pkgs: added ArangoDB 2.5.2

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

File: pkgs/servers/nosql/arangodb/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, openssl, zlib, python, gyp, bash, go, readline }:
+
+stdenv.mkDerivation rec {
+  version = "2.5.2";
+  name    = "arangodb-${version}";
+
+  src = fetchgit {
```
Comment (edwtjo) [inline pkgs/servers/nosql/arangodb/default.nix]: please use fetchFromGitHub instead

**Comment by lethalman** [comment]
Is it possible to share the v8 with the one built in nixpkgs? This would be yet another bundled v8 otherwise.

**Comment by flosse** [comment]
> Is it possible to share the v8 with the one built in nixpkgs? This would be yet another bundled v8 otherwise.

that's what I tried to do: https://github.com/arangodb/arangodb/issues/1316

**Comment by lethalman** [comment]
Thanks for your effort to convince the upstream. I see it's rather hardcoded in the configure... so let's add this for now though I'm unhappy yet another time. Built locally, looks fine thanks.

**Comment by wanggang1** [comment]
@flosse any plan to add a package for ArangoDB 3.x?

**Pull request merged**